docs/ref/models/options.txt

```diff
@@ -312,6 +312,12 @@ not be looking at your Django code. For example::
 
         ordering = [F("author").asc(nulls_last=True)]
 
+.. admonition:: Default ordering and GROUP BY
+
+    This default ordering is not used in :ref:`GROUP BY queries
+    <aggregation-ordering-interaction>`, such as those using
+    :meth:`~.QuerySet.annotate` and :meth:`~.QuerySet.values`.
```

Comment:
Let's reverse the order here. The doc'd examples usually show `values()` then `annotate()`, because the results can be subtly different otherwise.

Example:
https://github.com/django/django/blob/05ba1a9228128614fb3c475f1c4bdf0160f44dba/docs/ref/models/querysets.txt#L652-L653

Explanation: https://forum.djangoproject.com/t/group-by-and-having/41574/2